server/app/api.py

```diff
@@ -1498,6 +1504,15 @@ def exit(self, group, user, data):
             raise need.exception()
 
         group.exit(user)
+        
+    def save_order(self, group, user, data):
+        """ Saves order of partners """
+        need = Need('save_order')
+        if not group.can(user, need, group):
+            raise need.exception()
+
+        self.order = data['order']
+        self.save()
```

Comment:
Okay, so the logic here would go something like this:

new_order = data['order']

if len(new_order) != len(group.member):
    # Tell the frontend that they messed up

for member in group.member:
     if member not in new_order:
           # Tell the frontend they messed up again.

group.member = new_order